modules/apps/portal-search/portal-search-rest-impl/src/main/java/com/liferay/portal/search/rest/internal/resource/v1_0/SuggestionResourceImpl.java

```diff
@@ -217,6 +224,37 @@ else if (StringUtil.equals(scope, "this-site") && (groupId != null)) {
 		searchContext.setTimeZone(contextUser.getTimeZone());
 		searchContext.setUserId(contextUser.getUserId());
 
+		long commerceChannelGroupId = 0;
+
+		CommerceChannel commerceChannel =
+			_commerceChannelLocalService.fetchCommerceChannelBySiteGroupId(
+				groupId);
+
+		if (commerceChannel != null) {
+			commerceChannelGroupId = commerceChannel.getGroupId();
+		}
+
+		if (commerceChannelGroupId > 0) {
+			searchContext.setAttribute(
+				CPField.COMMERCE_CHANNEL_GROUP_ID, commerceChannelGroupId);
+			searchContext.setAttribute("secure", Boolean.TRUE);
+
+			AccountEntry accountEntry =
+				_commerceAccountHelper.getCurrentAccountEntry(
+					commerceChannelGroupId, themeDisplay.getRequest());
+
+			long[] commerceAccountGroupIds = null;
+
+			if (accountEntry != null) {
+				commerceAccountGroupIds =
+					_accountGroupLocalService.getAccountGroupIds(
+						accountEntry.getAccountEntryId());
+			}
+
+			searchContext.setAttribute(
+				"commerceAccountGroupIds", commerceAccountGroupIds);
```

Comment:
We need to also add `accountEntryId`.

```
					searchContext.setAttribute(
						"accountEntryId", accountEntry.getAccountEntryId());
```

Reply:
In fact, if we are fully committing to this, we need to pretty much all attributes from our [CPSearchResultsPortletSharedSearchContributor.java](https://github.com/liferay-commerce/liferay-portal/blob/672db6cf76c6187d46a93077b1440875b25e0891/modules/apps/commerce/commerce-product-content-search-web/src/main/java/com/liferay/commerce/product/content/search/web/internal/portlet/shared/search/CPSearchResultsPortletSharedSearchContributor.java#L117):

Reply:
![Highlighted](https://github.com/user-attachments/assets/18a2b1d3-61ba-43ce-9466-1274bbd3895f)

These are the minimum highlighted ones we should expect and align with if we want the results to reflect what would be returned if we only had `CPSearchResultsPortlet` on the page.